docs/explainer.md

```diff
@@ -77,18 +77,21 @@ appropriate rate of payments to make to the site.
 3. The browser generates a unique session ID for this payment session.
 4. The browser fetches a unique destination address and shared secret for the
```

Comment:
maybe this should be "from the site's WM receiver or Payment Verifier Service"

Reply:
Changed to "from site's payment pointer URL"
https://github.com/interledger/webmonetization.org/pull/51/commits/10a5fd13c2add0f347ba5e56a83fb87503998125